tests/test_modeling_t5.py

```diff
@@ -61,7 +61,7 @@ def __init__(
             initializer_factor=0.002,
             eos_token_id=1,
             pad_token_id=0,
-            decoder_start_token_id=0,
+            bos_token_id=0,
```

Comment:
Don't like this. T5 officially doesn't have a `bos_token_id`. If you check T5's tokenizer there is no `bos_token_id`. It might be confusing to initialize it here, but have different values in the tokenizer

Reply:
rolled back decoder_start_token_id changes